addons/mail/static/src/new/composer/suggestion_hook.js

```diff
@@ -3,7 +3,7 @@
 import { useComponent, useEffect, useState } from "@odoo/owl";
 import { useService } from "@web/core/utils/hooks";
 
-export function useSuggestion() {
+export function useSuggestion({ disabled = false }) {
```

Comment:
It doesn't really make sense to define a hook that is disabled, the hook should simply not be used in that case instead.